HousingSearchApi.Tests/V1/E2ETests/Stories/GetProcessesStories.cs

```diff
@@ -44,6 +44,15 @@ public void ServiceReturnsOkResult()
                 .BDDfy();
         }
 
+        [Fact]
+        public void ServiceReturnsOkResultWithOnlyTargetId()
```

Comment:
Shouldn't this be 400?

Reply:
Or does this method actually have both targetId and targetType?

Reply:
It has both, I'll update the test name